sdk/appconfiguration/azure-appconfiguration-provider/tests/test_async_provider_aad.py

```diff
@@ -23,7 +23,7 @@ async def test_provider_creation_aad(self, appconfiguration_endpoint_string, app
             assert client["my_json"]["key"] == "value"
             assert (
                 client["FeatureManagementFeatureFlags"]["Alpha"]
-                == '{"enabled": false, "conditions": {"client_filters": []}}'
+                == '{"id": "Alpha", "enabled": false, "conditions": {"client_filters": []}}'
```

Comment:
Just curious: Are these changes introduced  by feature management ?

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/31326